lib/templates/diff.eex

```diff
@@ -21,6 +21,8 @@
         </tr>
 
         <%= for {stop_name, stop_id_0, stop_id_1} <- stops do %>
+          <% stop_id_0 = {stop_id_0, 0} %>
```

Comment:
Do you need to make a change in the non-diff template as well?

Reply:
I'll check: I'll confess don't use that view as much.